Makefile

```diff
@@ -137,7 +137,7 @@ EKS_A_CROSS_PLATFORMS := $(foreach platform,$(EKS_A_PLATFORMS),eks-a-cross-platf
 E2E_CROSS_PLATFORMS := $(foreach platform,$(EKS_A_PLATFORMS),e2e-cross-platform-$(platform))
 EKS_A_RELEASE_CROSS_PLATFORMS := $(foreach platform,$(EKS_A_PLATFORMS),eks-a-release-cross-platform-$(platform))
 
-DOCKER_E2E_TEST := TestDockerKubernetes125SimpleFlow
+DOCKER_E2E_TEST := TestDockerKubernetes126ManagementClusterUpgradeFromLatestSideEffects
```

Comment:
Do we want to keep this change?

Reply:
No I'm just testing. I have the hold label on the PR